integration/conftest.py

```diff
@@ -96,28 +91,22 @@ def reactor():
 
 
 @pytest.fixture(scope='session')
-@log_call(action_type=u"integration:temp_dir", include_args=[])
-def temp_dir(request):
+@log_call(action_type=u"integration:base_dir", include_args=[])
+def base_dir(request, tmp_path_factory):
```

Comment:
It might be worth documenting that `tmp_path_factory` (and `request`) are magic pytest fixtures since it may not be obvious to all readers all the time.  A link to the docs might be nice too, although I didn't have any trouble looking up `tmp_path_factory` just based on the name.